critiquebrainz/db/review.py

```diff
@@ -628,3 +628,29 @@ def distinct_entities():
               FROM review
         """))
         return {row[0] for row in results.fetchall()}
+
+
+def reviewed_entities(*, entity_ids, entity_type):
```

Comment:
This function name is a little misleading, we should either rename it to `check_if_entities_reviewed` or something along those lines or keep the name and make it return a list of the gids of those entities which have been reviewed (which we're doing in `get_entities_by_gid` right now), I kinda prefer the second method.

Reply:
Also, we should add a test for this function as well.

Reply:
I've added a test for the function `reviewed_entities`.